### ISSUE TITLE
Mget file failure on file download (COS -> CVM)

### ISSUE THREAD

**Issue by reinhouthooft** [tencentyun/coscmd#173]
```
Download cos://rein-houthooft/exp_data/lf_dict_501_310.json   =>   /home/ubuntu/data/exp_data/lf_dict_501_310.json
Downloading /home/ubuntu/data/exp_data/lf_dict_501_310.json

  0%|          | 0.00/1.59M [00:00<?, ?B/s][Errno 17] File exists: '/home/ubuntu/data/exp_data'

 50%|#####     | 793K/1.59M [00:00<00:00, 6.88MB/s]
 90%|######### | 1.43M/1.59M [00:00<00:00, 10.1MB/s]
Completing mget
[Errno 2] No such file or directory: u'/home/ubuntu/data/exp_data/lf_dict_501_310.json_5'
Mget file failure
```
Getting this error sometimes, could you investigate?

**Comment by lewzylu** [comment]
Does that mistake happen every time?

**Comment by reinhouthooft** [comment]
@lewzylu Only sometimes (I can't find any pattern about when it happens exactly, seems random).

**Comment by reinhouthooft** [comment]
Also, it _only_ happens with that specific file.

**Comment by bradelement** [comment]
![wechatimg117](https://user-images.githubusercontent.com/1546657/44075676-3e27c5a2-9fd0-11e8-8cb7-31b7b9343ef1.png)

I also got a similar problem. The failed file is `model.aqbin` 

sometimes it says model.aqbin_5 is missing, sometimes it says model.aqbin_1 is missing.

**Comment by lewzylu** [comment]
Maybe threr is a bug of  multidownload, I’ll look at it in the next few days.

**Comment by lewzylu** [comment]
it's a bug of function ‘makedirs’ which has been fixed in 1.8.5.5.
Thx for your advice
